xds/internal/balancer/clusterresolver/configbuilder.go

```diff
@@ -234,12 +248,97 @@ func dedupSortedIntSlice(a []int) []int {
 	return a[:i+1]
 }
 
+// priorityLocalitiesToClusterImpl takes a list of localities (with the same
+// priority), and generates a cluster impl policy config, and a list of
+// addresses.
+func priorityLocalitiesToClusterImpl(localities []xdsclient.Locality, priorityName string, mechanism DiscoveryMechanism, drops []clusterimpl.DropConfig, xdsLBPolicy *internalserviceconfig.BalancerConfig) (*clusterimpl.LBConfig, []resolver.Address, error) {
+	var addrs []resolver.Address
```

Comment:
Nit: Do we need to declare this variable here? Seems like each of those `if` statements can have their own local copy.

Reply:
Right. Done